app/views/service_calendars/master_calendar/pppv/_check_visit_columns.html.haml

```diff
@@ -20,7 +20,7 @@
 - visit_groups.each_with_index do |visit_group, index|
   %th.col-sm-1.text-center
     - if tab == 'template'
-      = Dashboard::ServiceCalendars.select_column(visit_group, index, portal, service_request, sub_service_request)
+      = Dashboard::ServiceCalendars.select_column(visit_group, (page - 1) * Visit.per_page + index, portal, service_request, sub_service_request)
```

Comment:
https://github.com/sparc-request/sparc-request/blob/jjh-for-readability/lib/dashboard/service_calendars.rb#L163